packages/sogrim-app/src/components/Banner/BannerDialogs/ImportGilion.tsx

```diff
@@ -102,6 +116,12 @@ export const ImportGilion: React.FC<ImportGilionProps> = ({
           autoFocus
           fullWidth
           margin="dense"
+          error={isError}
+          helperText={
+            isError
+              ? "ייבוא גיליון הציונים כשל. האם העתקתם את כל גיליון הציונים (ולא תעודת ציונים!) , דרך דפדפן כרום?"
```

Comment:
"דרך דפדפן כרום או אדג'".
and because "אדג'" looks a little bit weird I would prefer:
דרך דפדפן chrome או edge.